Makefile

```diff
@@ -88,19 +88,24 @@ check_flags =
 ifeq ($(FIREDRAKE_CI_TESTS), 1)
 	check_flags = --verbose
 else
-	check_flags = --quiet --no-summary
+	check_flags = --quiet
+endif
+
+timeout_expr =
+ifdef TIMEOUT
+	timeout_expr = --timeout $(TIMEOUT)
 endif
 
 .PHONY: check
 check:
 	@echo "    Running serial smoke tests"
-	@python -m pytest $(check_flags) \
+	@python -m pytest $(check_flags) $(timeout_expr) \
```

Comment:
change `$(timeout_expr)` to `$(PYTEST_ARGS_CHECK)` so we aren't restricting what arguments we can pass to make check without modifying the makefile.